### ISSUE TITLE
Download error (non fatal) in the TCL plugin

### ISSUE THREAD

**Issue by chr15m** [pure-data/deken#4]
```
bad option "status": must be -code, -errorcode, or -errorinfo
bad option "status": must be -code, -errorcode, or -errorinfo
    while executing
"return status errorstatus"
    (procedure "dialog_externals_search::download_file" line 12)
    invoked from within
"dialog_externals_search::download_file $URL $destination"
    (procedure "dialog_externals_search::clicked_link" line 5)
    invoked from within
"dialog_externals_search::clicked_link .externals_searchui
http://puredata.info/Members/fjkraan/cyclone_0.1_alpha57-xtrnl-Darwin-ppc-i386-32bit-x86_64-..."
    (command bound to event)
```

**Comment by umlaeute** [comment]
fixed with 27738985f49020e1abc1b9a59207688317cffd0d (pull request #20)

**Comment by chr15m** [comment]
@electrickery said: "Yes, with the latest deken-plugin.tcl download and install on the mac is
now without problem."
